ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/vm/GeneralStateReferenceTestTools.java

```diff
@@ -206,4 +227,33 @@ public static void executeTest(final GeneralStateTestCaseEipSpec spec) {
   private static boolean shouldClearEmptyAccounts(final String eip) {
     return !SPECS_PRIOR_TO_DELETING_EMPTY_ACCOUNTS.contains(eip);
   }
+
+  private static void logWorldState(final ReferenceTestWorldState worldState) {
+    ObjectMapper mapper = new ObjectMapper();
+    ObjectNode worldStateJson = mapper.createObjectNode();
+    worldState.streamAccounts(Bytes32.ZERO, Integer.MAX_VALUE)
+        .forEach(
+            account -> {
+              ObjectNode accountJson = mapper.createObjectNode();
+              accountJson.put("nonce", Bytes.ofUnsignedLong(account.getNonce()).toShortHexString());
+              accountJson.put("balance", account.getBalance().toShortHexString());
+              accountJson.put("code", account.getCode().toHexString());
+              ObjectNode storageJson = mapper.createObjectNode();
+              var storageEntries = account.storageEntriesFrom(Bytes32.ZERO, Integer.MAX_VALUE);
+              storageEntries.values().stream()
+                  .map(
+                      e ->
+                          Map.entry(
+                              e.getKey().orElse(UInt256.ZERO),
+                              account.getStorageValue(UInt256.fromBytes(e.getKey().get()))))
+                  .sorted(Map.Entry.comparingByKey())
+                  .forEach(e -> storageJson.put(e.getKey().toQuantityHexString(), e.getValue().toQuantityHexString()));
+
+              if (!storageEntries.isEmpty()) {
+                accountJson.set("storage", storageJson);
+              }
+              worldStateJson.set(account.getAddress().orElse(Address.ZERO).toHexString(), accountJson);
```

Comment:
same here, I don't think we will trigger this case but I will prefer to see that it was empty and no think that the address is the ZERO one